tests/e2e/scale/test_scale_12_OCS_worker_nodes_and_6000_PVCs.py

```diff
@@ -75,15 +75,16 @@ def test_scale_node_and_capacity(self):
                 scale_worker_count = expected_worker_count - len(
                     existing_ocs_worker_list
                 )
-                assert scale_lib.scale_ocs_node(node_count=scale_worker_count)
+                scale_lib.scale_ocs_node(node_count=scale_worker_count)
```

Comment:
why remove the assert. would it work for all platforms and scenarios?

Reply:
Removed assert to handle the failure in exception. With assert, exception code is not reachable. In exception code there is a flag `TestAddNode.skip_all = True` which is used in all the testcase to decide either to skip the test or not. 
i.e. In-case of failures in test_scale_node_and_capacity() testcase, exception will set `TestAddNode.skip_all = True` and following tests will be skipped.

Reply:
Makes sense. Thank you for the explanation Ram.